packages/paste-core/components/avatar/src/Avatar.tsx

```diff
@@ -123,11 +123,13 @@ export const Avatar = React.forwardRef<HTMLDivElement, AvatarProps>(
         flexShrink={0}
         ref={ref}
         size={groupSize || size}
-        {...(src ? undefined : shadowVariants[color])}
+        {...(src ? {boxShadow: 'shadowBorderWeaker'} : shadowVariants[color])}
         {...colorVariants[color]}
         {...variants[groupVariant || variant]}
+        // when its an image the background should be transparent, for when you are using transparent images
+        {...(src && {backgroundColor: 'transparent'})}
```

Comment:
praise: good fix! 

question: can this be merged into line 126 or is the order in which these styles applied brittle?

Reply:
Yeah, I don't love this. Imma rethink this logic a bit, it feels too much like I'm shoe-horning this in

Reply:
OK, this now feels much better